x/evm/precompile/funtoken.go

```diff
@@ -69,15 +69,15 @@ func (p precompileFunToken) Run(
 	method := startResult.Method
 	switch PrecompileMethod(method.Name) {
 	case FunTokenMethod_sendToBank:
-		bz, err = p.sendToBank(startResult, contract.CallerAddress, readonly)
+		bz, err = p.sendToBank(startResult, contract.CallerAddress, readonly, evm)
```

Comment:
We pass in the `evm` object to precompile methods that need access to `StateDB`.

Reply:
Yes, this looks so much cleaner and less error-prone